dao/README.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Orochi DAO
+---
+
+# Orochi DAO - event sponsorship as a DAO
+
+## What is Orochi DAO?
+
+<p class="companies">
+    <a href="https://orochi.kickback.events">
+        <img src="/orochi.png">
+    </a>
+</p>
+
+
+There will be many side events during DevCon5 and many companies will be sponsoring various side events to reach out to the participants. However, sponsoring the individual event will have limited marketing impact. Kickback, the organiser of “[The year of DAOs](/events/the-year-of-daos/)” event would like to propose “Orochi DAO”, a special purpose DAO to coordinate sponsorship allocation collaboratively.
+
+Orochi DAO is a DAO which manages event sponsorship. It is a fork of [Moloch DAO](https://molochdao.com) which was created to fund ETH 2.0 projects and famously collected over ETH 7000 from well known Ethereum community members.
+ The sponsoring companies (= Donors in Moloch term) and the event organisers (= Grantees in Moloch term) all join the DAO based on the proposal from the existing members (= Guild members in Moloch term) . The DAO members will collaboratively decide which events to sponsor for how much.
+
+By using “Rage quit” feature of Moloch DAO, the sponsoring companies can withdraw their sponsorship when they do not agree with the voting results. With this simple game theory, we are experimenting to see if each event organiser and sponsor can work collaboratively to have maximum PR impact as a whole. 
+
+## What is “The Year of DAOs”?
+
+We are aiming to invite speakers from well known DAO framework creators and the actual users of these frameworks so that we will learn the practical use of DAOs.
+
+We are currently in the process of building up the program at our [public forum page](https://forum.metacartel.org/t/orochidao-call-for-ideas-of-the-year-of-daos-event-content/179/).
+
+Please check “[our event calendar page](/events/the-year-of-daos/)” for more detail.
+
+## Eligibility
+
+To apply to the guild member as a sponsor, the proposal must be submitted by [the existing guild members](https://daohaus.club/dao/0x8487dcc6f4b28b911e22a8657ebb16427d4cf5c0)(To see member profiles, you need to connect via dapp wallets such as Metamask). The minimum participation contribution is $500 although the decisions will be made by the DAO voting hence it may vary case by case. The contribution will be made [in DAI](https://makerdao.com/en/dai/).
```

Comment:
```suggestion
To apply to the DAO as a sponsor, the proposal must be submitted by [the existing guild members](https://daohaus.club/dao/0x8487dcc6f4b28b911e22a8657ebb16427d4cf5c0)(To see member profiles, you need to connect via dapp wallets such as Metamask). The minimum sponsorship is $500, although final decisions will be made by the DAO members' voting, hence it may vary case by case. The contribution will be made [in DAI](https://makerdao.com/en/dai/).
```